pimd/subdir.am

```diff
@@ -59,6 +58,7 @@ pim_common = \
 
 pimd_pimd_SOURCES = \
 	$(pim_common) \
+	pimd/pim_bsm.c \
```

Comment:
The BSM code doesn't work yet in this PR?

Reply:
Few changes are done in 10952, therefore I have not repeated it here. Therefore had to add in stub to keep compilation in working condition.

Reply:
Also the BSM changes depends on this RP changes, therefore once this PR gets merged, 10952 will update and remove this part of code.